.clippy.toml

```diff
@@ -1 +1 @@
-msrv = "1.58.0"
+msrv = "1.74.0"
```

Comment:
The project currently does not compile with version 1.73.0. I could look at fixing this if you would like to keep the msrv lower.

Reply:
I think it's fine to set 1.74 as minimum version